Tasks/AzurePowerShell/AzurePowerShell.ps1

```diff
@@ -6,6 +6,8 @@ $scriptType = Get-VstsInput -Name ScriptType -Require
 $scriptPath = Get-VstsInput -Name ScriptPath
 $scriptInline = Get-VstsInput -Name Inline
 $scriptArguments = Get-VstsInput -Name ScriptArguments
+$targetAzurePs = Get-VstsInput -Name TargetAzurePs
```

Comment:
add logging for targetAzurePs  and customTargetAzurePs  if Get-VstsInput is not doing